### ISSUE TITLE
Run in a non-interactive way

### ISSUE THREAD

**Issue by giulia-berto** [HALFpipe/HALFpipe#108]
Hi! Is it possible to run HALFpipe non-interactively?

**Comment by HippocampusGirl** [comment]
Yes! You need to generate a `spec.json` file and place it in an empty directory. Then run HALFpipe the normal way, but add the options `--skip-spec-ui --workdir $(pwd)`. This will tell HALFpipe not to open the user interface, and where to look for the `spec.json`. 

To generate a `spec.json` file, I would advise to try to make one as close as possible to what you want to do in the interactive way, and then edit the JSON syntax manually to get what you need. 

Finally, you can also use the `--use-cluster` option, which will generate a non-interactive submission script for HPC.

**Comment by giulia-berto** [comment]
Awesome, thanks @HippocampusGirl, will try!
